### PR TITLE
related to issue  #111: google-map-search found no results

### PR DESCRIPTION
**Pull request by supita**

### CODE REVIEW

**Comment by supita** [comment]
CLA signed

File: google-map-search.html [hunk 1]
```diff
@@ -82,12 +82,14 @@
...
-          latitude: results[0].geometry.location.lat(),
-          longitude: results[0].geometry.location.lng(),
-          show: true
+        if (status == google.maps.places.PlacesServiceStatus.OK) {
+          this.result = {
+            latitude: results[0].geometry.location.lat(),
+            longitude: results[0].geometry.location.lng(),
+            show: true
+          };
+          this.fire('google-map-search-result', this.result);
```
Comment (ebidel) [inline google-map-search.html]: This is an API change where the element no longer fires the event if the status wasn't OK. I think we should instead still fire the event but `this.result` is null

**Comment by supita** [comment]
I understand, I was just trying to follow what you guys did in the [route](https://github.com/GoogleWebComponents/google-map/blob/623a55ecf75e839b1b0ade0aa4100854659ce308/google-map-directions.html#L152-L157) function of google-map-directions. Also the documentation says

>  @event google-map-search-result 
> Fired when the search element returns a result

and as there was no result, I thought the event shouldn't be fired

**Comment by ebidel** [comment]
I think the description was incorrect to begin with. It should really read "Fired when the directions API service returns a response"

**Comment by ebidel** [comment]
Thanks for the PR! I'm going to close this since it's pre Polymer 1.0. If you'd like to resubmit a refresh PR for the update 1.0 element, please do!

**Pull request closed without merging**